package.json

```diff
@@ -99,6 +99,7 @@
     "fast-glob": "3.2.12",
     "ignore": "5.2.4",
     "is-glob": "4.0.3",
+    "jest-diff": "^28.1.3",
```

Comment:
why do you need this package?

Reply:
Ops, good catch! I used it to try to debug and diff the mismatching `CODEOWNERS` files on the Windows runners. I will remove it! Good eyes! :star2: